packages/migrate/src/__tests__/Baseline.test.ts

```diff
@@ -0,0 +1,115 @@
+      Prisma Migrate created the following migration without applying it 20201231000000_
+
+      You can now edit it and apply it by running prisma migrate dev.
+    `)
+    expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(`
+      Prisma schema loaded from prisma/schema.prisma
+      Datasource "my_db": SQLite database "dev.db" at "file:./dev.db"
+
+      Drift detected: Your database schema is not in sync with your migration history.
+
+      The following is a summary of the differences between the expected database schema given your migrations files, and the actual schema of the database.
+
+      It should be understood as the set of changes to get from the expected schema to the actual schema.
+
+      If you are running this the first time on an existing database, please make sure to read this documentation page:
+      https://www.prisma.io/docs/guides/database/developing-with-prisma-migrate/troubleshooting-development
+
+      [+] Added tables
+        - Blog
+
+
+    `)
+    ctx.mocked['console.info'].mockReset()
+
+    // migrate dev
+    const migrateDev = MigrateDev.new().parse([])
+    await expect(migrateDev).resolves.toMatchInlineSnapshot(``)
+    expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(`
+      Prisma schema loaded from prisma/schema.prisma
+      Datasource "my_db": SQLite database "dev.db" at "file:./dev.db"
+
+      Applying migration \`20201231000000_\`
+
+      The following migration(s) have been applied:
+
+      migrations/
+        └─ 20201231000000_/
+          └─ migration.sql
+
+      Your database is now in sync with your schema.
+    `)
+    ctx.mocked['console.info'].mockReset()
+
+    // Switch to PROD database
+    process.env.DATABASE_URL = 'file:./prod.db'
+
+    // migrate resolve --applied migration_name
+    const migrationName = fs.list('prisma/migrations')![0]
+    const migrateResolveProd = MigrateResolve.new().parse(['--applied', migrationName])
+    await expect(migrateResolveProd).resolves.toMatchInlineSnapshot(`Migration 20201231000000_ marked as applied.`)
+
+    expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(`
+      Prisma schema loaded from prisma/schema.prisma
+      Datasource "my_db": SQLite database "prod.db" at "file:./prod.db"
+    `)
+    ctx.mocked['console.info'].mockReset()
+
+    // migrate deploy
+    const migrateDeployProd = MigrateDeploy.new().parse([])
+    await expect(migrateDeployProd).resolves.toMatchInlineSnapshot(`No pending migrations to apply.`)
+    expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(`
+      Prisma schema loaded from prisma/schema.prisma
+      Datasource "my_db": SQLite database "prod.db" at "file:./prod.db"
+
+      1 migration found in prisma/migrations
+
+
+    `)
+
+    expect(ctx.mocked['console.log'].mock.calls).toEqual([])
+    expect(ctx.mocked['console.error'].mock.calls).toEqual([])
+  })
```

Comment:
If we are only testing what the commands would print, I don't know if we print anything, but shouldn't there be a message about no migrations having been run?

Reply:
So the message you are looking for is line 102 `No pending migrations to apply.`
If we switched to mocking `stdout` we would get all the prints in one place.

Reply:
:+1: